src/boss/cloudvm/worker_config_template.go

```diff
@@ -1,42 +1,49 @@
 package cloudvm
 
 import (
-	"fmt"
-	"os"
+	"encoding/json"
+	"log"
 	"path/filepath"
 
 	"github.com/open-lambda/open-lambda/ol/common"
 )
 
-func LoadWorkerConfigTemplate(templatePath string, workerPath string) error {
-	// Create template.json if it doesn't exist
-	if _, err := os.Stat(templatePath); os.IsNotExist(err) {
-		// If template.json does not exist, load defaults
-		if err := common.LoadDefaults(workerPath); err != nil {
-			return fmt.Errorf("failed to load defaults: %v", err)
-		}
+func SaveTemplateConfToWorkerDir(cfg *common.Config, workerPath string, workerPort string) error {
```

Comment:
We're constructing a new config based on 3 pieces of info:
1. the template
2. worker default
3. next port number

This is a bit complex so there should be comments explaining what we're doing.